pages.es/osx/krunvm.md

```diff
@@ -0,0 +1,24 @@
+# krunvm
```

Comment:
Warning: this page contains 1 or more mandatory to fix issues (source: original page).